doc/README_CN.md

```diff
@@ -16,59 +16,62 @@
 (_______)_______/  )_(  |/     \|   v0.10
 ```
 
-# Git 啊：管理多个 git 库的命令行工具
+# Gita：一个管理多个 git 库的命令行工具
 
-老夫有两把刷子：
+这个工具有两个作用：
 
-- 同时显示多个库的状态信息，比如库杈名，编辑状态，贡献信息等
+- 并排显示多个库的状态信息，比如分支名，编辑状态，提交信息等
 - 在任何目录下代理执行 git 指令
 
 ![gita screenshot](https://github.com/nosarthur/gita/raw/master/doc/screenshot.png)
 
-库杈的五色代表了本土和远程库杈相生相杀的关系：
+本地和远程分支之间的关系有5种情况，在这里分别用5种颜色对应着：
 
-- 青：远近一致
-- 赤：本土已与远程分道扬镳
-- 黄：本土落后于远程（宜和合merge）
-- 白：没有远程
-- 紫：本土超前于远程（宜推送push）
+- 绿色：本地和远程保持一致
+- 红色：本地和远程产生了分叉
+- 黄色：本地落后于远程（适合合并）
+- 白色：本地没有指定远程
+- 紫色：本地超前于远程（适合推送）
 
-如果你知道蓝移和红移的道理，就会明白为什么黄色表示落后，紫色表示超前。
+为什么选择了紫色作为超前以及黄色作为落后，绿色作为基准 的理由在这两篇文章中解释：  
+[blueshift](https://en.wikipedia.org/wiki/Blueshift)、[redshift](https://en.wikipedia.org/wiki/Redshift)
 
-编辑状态符号：
+额外的状态符号意义：
 
-- `+`: 已上台(staged)
-- `*`：未上台(unstaged)
-- `_`：未追踪(untracked)
+- `+`: 暂存(staged)
+- `*`： 未暂存（unstaged）
+- `_`： 未追踪（untracked）
 
-基础指令：
+记录的子命令：
 
-- `gita add <repo-path(s)>`: 为老夫添加库
-- `gita rm <repo-name(s)>`: 取消库（不会删除文件）
-- `gita ll`: 显示所有库的信息
+- `gita add <repo-path(s)>`: 添加库
+- `gita rm <repo-name(s)>`: 移除库（不会删除文件）
+- `gita ll`: 显示所有库的状态信息
 - `gita ls`: 显示所有库的名字
 - `gita ls <repo-name>`: 显示一个库的绝对路径
 - `gita rename <repo-name> <new-name>`: 重命名一个库
-- `gita info`: 显示已用的和未用的信息项
-- `gita -v`: 显示老夫的版本号
+- `gita info`: 显示可用的、是否在使用的信息项目
```

Comment:
I feel the original translation is more correct.

Reply:
- `gita info`: display the information items available and in use
it's two describe words : available , in use.
In mine mind, `in use` is a smaller scope of `available` : when available , it can be in use or not in use. if not available, it of course is not in use.

Reply:
so the output of this sentence, as i thought, must be available -- in use or not in use.
with no unavailable info

Reply:
I see the discrepancy now. The README.md is not up to date. If you try `gita info`, you can see it's really in-use and unused. I will update README.md

Reply:
updated

Reply:
Now is the original better.
And these diff, should i push again or u can just ignore?

Reply:
You can push again, thanks